app/controllers/docuseal_controller.rb

```diff
@@ -34,6 +34,12 @@ def webhook
         document.save!
         contract.update(document:)
         contract.mark_signed!
+
+        # Unfreeze the event if this is the first signed contract
+        event = contract.organizer_position_invite.event
+        if event.organizer_position_contracts.signed.count == 1
+          event.update!(financially_frozen: false)
+        end
```

Comment:
Should this be a callback of `mark_signed!`? It feels a tad bit cleaner that way

Reply:
nice - yeah - moved!